includes/class-sensei-quiz.php

```diff
@@ -1223,14 +1223,13 @@ public static function  the_user_status_message( $quiz_id ){
 
         $lesson_id =  Sensei()->quiz->get_lesson_id( $quiz_id );
         $status = Sensei_Utils::sensei_user_quiz_status_message( $lesson_id , get_current_user_id() );
-        $message = '<div class="sensei-message ' . $status['box_class'] . '">' . $status['message'] . '</div>';
         $messages = Sensei()->frontend->messages;
 
-        if ( !empty( $messages ) ) {
-          $message .= $messages;
+        if ( ! empty( $messages ) ) {
```

Comment:
This logic is not equivalent to what it was. If `$messages` is not empty, it would concatenate the messages onto the single status message. Now if it is not empty, it only displays the `$messages`.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/2296/commits/d190f25de2df93a75cb6fe3eb98bc48f8147606e.